Geometry

hyunyong opened the pull request.

hyunyong committed:
Update muonNumbering.xml

hyunyong committed:
Update README.md

jshlee commented on README.md:
```diff
@@ -1,8 +1,12 @@
-cmsrel CMSSW_10_5_0_pre1
-cd CMSSW_10_5_0_pre1/src
+cmsrel CMSSW_10_5_0_pre2
+cd CMSSW_10_5_0_pre2/src
```
i would prefer CMSSW_10_6_0X

hyunyong committed:
Update README.md

I've done a test with 10_6_0_pre3.
10_6_0_pre3 doesn't have vfatv3 unpacker yet.

hyunyong committed:
Merge branch 'master' into geometry
